File: dataset/output/GEMINI_gemini-1.5-pro-001/Task70_GEMINI_gemini-1.5-pro-001.java
import java.io.IOException;
import java.nio.file.Files;
import java.nio.file.Paths;

public class ReadFileContent {
    public static void main(String[] args) throws IOException {
        if (args.length == 0) {
            System.err.println("Please provide the file path as a command line argument.");
            System.exit(1);
        }
        String filePath = args[0];
        String content = new String(Files.readAllBytes(Paths.get(filePath)));
        System.out.print(content);
    }
}

File: dataset/output/GEMINI_gemini-1.5-pro-001/Task70_GEMINI_gemini-1.5-pro-001.cpp
#include <iostream>
#include <fstream>
#include <string>

using namespace std;

int main(int argc, char* argv[]) {
  if (argc < 2) {
    cerr << "Error: Please provide the file path as a command-line argument." << endl;
    return 1;
  }

  string filePath = argv[1];
  ifstream file(filePath);

  if (file.is_open()) {
    string line;
    while (getline(file, line)) {
      cout << line << endl;
    }
    file.close();
  } else {
    cerr << "Error: Unable to open file: " << filePath << endl;
    return 1;
  }

  return 0;
}